.gitignore

```diff
@@ -13,4 +13,5 @@ __pycache__
 .eggs
 .pytest_cache
 lin.cpp
-bazel-*
\ No newline at end of file
+bazel-*
+compile_commands.json
```

Comment:
This was for use with the same `bazel` compilation intellisense extension used in `psim`.